nncf/onnx/quantization/quantizer_parameters.py

```diff
@@ -51,96 +78,37 @@ def get_level_low_level_high(tensor_type: np.dtype, num_bits: int) -> Tuple[int,
     return - (2 ** num_bits) // 2, (2 ** num_bits) // 2 - 1
 
 
-def calculate_scale_zero_point(max_val: np.ndarray, min_val: np.ndarray, level_low: int, level_high: int,
-                               mode: QuantizationMode, tensor_type: np.dtype) -> Tuple[np.ndarray, np.ndarray]:
+def calculate_scale_zero_point(input_low: np.ndarray, input_high: np.ndarray, level_low: int, level_high: int,
+                               mode: QuantizationMode, eps: float = 1e-6) -> Tuple[np.ndarray, np.ndarray]:
     """
     Calculates Quantizer/Dequantizer layer scale level.
-    Returns scale and zero_point values foe the quantizer.
-
-    :param max_val: The maximum values of the input tensor.
-    :param min_val: The minimum values of the input tensor.
-    :param level_low: The minimum level of quantizer quants.
-    :param level_high: The maximum level of quantizer quants.
+    Returns scale and zero_point values for the quantizer.
+
+    :param input_low: The minimum limit for an input value based on collected statistics.
+    :param input_high: The maximum limit for an input value based on collected statistics.
+    :param level_low: The minimum level in the integer range to quantize.
+        The default is "0" for an unsigned range, and "-2^(bit-1)" for a signed one .
+    :param level_high: The maximum level in the integer range to quantize.
+        The default is "2^bits-1" for an unsigned range, and "2^(bit-1)-1" for a signed one.
     :param mode: Symmetric or Asymmetric mode.
-    :param tensor_type: Value type of the tensor. Could be INT8 or UINT8.
+    :param eps: The correction value for scale to avoid division by zero.
     :return: Scale and Zero point values.
     """
     scale, zero_point = None, None
     if mode == QuantizationMode.SYMMETRIC:
-        input_abs_max = np.maximum(np.abs(max_val), np.abs(min_val))
-        max_val = input_abs_max
-        min_val = np.zeros_like(input_abs_max) if tensor_type == np.uint8 else -input_abs_max
-        scale = np.array((max_val - min_val) / float(level_high - level_low))
+        scale = np.array((input_high - input_low) / (level_high - level_low))
         zero_point = np.zeros_like(scale, dtype=np.int32)
     elif mode == QuantizationMode.ASYMMETRIC:
-        scale = np.array((max_val - min_val) / float(level_high - level_low))
-        zero_point = np.round(level_low - min_val / scale).astype(np.int32)
+        scale = np.array((input_high - input_low) / (level_high - level_low))
+        zero_point = np.round(level_low - input_low / np.maximum(scale, eps)).astype(np.int32)
```

Comment:
Usually we use scale + eps to make this calculations safe. Is it ok to use `np.maximum` here?

Reply:
I use `maximum`, because `scale` always >= 0 and `eps` only will be in denominator if `scale` is too close to zero, otherwise we will use original `scale`.

Reply:
Ok, got it. My concern is that ONNX implementation differs from Torch implementation in term of `eps` usage